azure-devops/idpay-projects/07_pipeline_idpay_portal_welfare_frontend.tf

```diff
@@ -11,6 +11,7 @@ variable "idpay-portal-welfare-frontend" {
     pipeline = {
       enable_code_review = true
       enable_deploy      = true
+      path               = "idpay\\idpay-portal-welfare-frontend"
```

Comment:
Please use this variable inside modules, or notthing will happen.

And use the module version 2.2.0, in this module you can find the new attribute path